kotlin/build.gradle

```diff
@@ -2,15 +2,14 @@
 apply from: '../version.gradle'
 
 buildscript {
-
     apply from: '../version.gradle'
     repositories {
         google()
         jcenter()
     }
     dependencies {
         classpath "com.android.tools.build:gradle:$gradleVersion"
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"}
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion" }
```

Comment:
I don't think this makes a blind bit of difference, but let's remove the added space so there are no changes being pushed into the main build.gradle script

Reply:
yes!!